src/tictactoe/Space.java

```diff
@@ -15,6 +15,14 @@ public Space(int row, int col) {
         this.col = col;
     }
 
+    public boolean isSame(Space space) {
+        if (this.row == space.row && this.col == space.col) {
```

Comment:
Remove `if else` block change to `return(this.row == space.row && this.col == space.col)`